build.sbt

```diff
@@ -14,7 +14,8 @@ incOptions := incOptions.value.withNameHashing(true)
 
 libraryDependencies ++= Seq(
     "com.kinja" %% "amqp-client" % "1.5.0",
-    "com.typesafe.play" %% "play-json" % "2.3.4"
+    "com.typesafe.play" %% "play-json" % "2.3.4",
+    "com.typesafe.slick" %% "slick" % "1.0.1"
```

Comment:
As discussed, let's keep this implementation in this library for now (as otherwise it would add more complexity to version changes) and create different slick branches (versions) if needed.